.github/workflows/e2e-main.yml

```diff
@@ -148,10 +148,6 @@ jobs:
       env:
         TERM: xterm
 
-    - name: Generate Mochawesome Reports
-      if: failure()
```

Comment:
This meant - if workflow fails for any reason and it led to errors in rare circumstances.